kolibri/core/assets/src/core-getters.js

```diff
@@ -32,6 +32,10 @@ function getDefaultChannelId(channelList) {
 
 /* return the current channel object, according to vuex state */
 function getCurrentChannelObject(state) {
```

Comment:
I'm confused why this would return `true` when there are no channels?

Previously, the function returned either a channel object, or `undefined`. (see [`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) docs.)

The existing logic would handle the case where `state.core.channels.list` is an empty array because calling `find` on `[]` will always return `undefined`. So it sounds like possibly the problem was that `state.core.channels.list` was undefined? If that was the case, we could do something like

```javascript
// if no channels to match against, return true
if (state.core.channels.list === undefined) {
  return undefined;
}
```

But that would still return the opposite of what's here - `undefined` instead of `true`...